auction/program/src/processor/create_auction.rs

```diff
@@ -99,11 +100,65 @@ pub fn create_auction(
         WinnerLimit::Unlimited(_) => BASE_AUCTION_DATA_SIZE,
     };
 
+    let bid_type_u64: u64 = match bid_type {
+        Some(v) => v as u64,
+        None => 0,
+    };
+
     let bid_state = match args.winners {
-        WinnerLimit::Capped(n) => BidState::new_english(n),
+        WinnerLimit::Capped(n) => match bid_type_u64 {
+            0 => BidState::new_english(n),
+            2 => BidState::new_dutch(n),
+            _ => BidState::new_english(n),
+        },
         WinnerLimit::Unlimited(_) => BidState::new_open_edition(),
     };
 
+    let mut decline_interval = 0;
+    let lamp = 1000000000;
+    let decline_rate = 2 * lamp;
+    let mut send_decrease_rate: u64 = 0;
```

Comment:
decreaes rate and decrease interval was going to be custom, using the params passed in but was hard coded instead.